src/path/fs.rs

```diff
@@ -11,11 +11,14 @@ use std::fs;
 use std::io::{self, Read};
 use std::path;
 
+use std::path::PathBuf;
+use std::path::Path;
```

Comment:
While I know its common practice in Rust, I personally don't like `use`ing concrete types.

The more interesting aspect is the inconsistency, there is both "use std::path" and using of concrete types.  We should only do one or the other.